packages/noco-docs/docs/010.index.md

```diff
@@ -4,7 +4,8 @@ description: 'NocoDB Documentation'
 slug: /
 ---
 
-## Welcome!
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
```

Comment:
I'd suggest to register these to global scope so that you don't need to import them to each MD file one by one.

To do this, you can create `theme/MDXComponents.js`. 

```js
// import the original mapper
import MDXComponents from '@theme-original/MDXComponents';
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
// some custom components goes here ...


export default {
  // re-use the default mapping
  ...MDXComponents,
  // some custom components goes here ...
  Tabs,
  TabItem,
};
```